Insertion Sort/README.markdown

```diff
@@ -91,15 +91,15 @@ Here is an implementation of insertion sort in Swift:
 
 ```swift
 func insertionSort(_ array: [Int]) -> [Int] {
-    var a = array			 // 1
-    for x in 1..<a.count {		 // 2
-        var y = x
-        while y > 0 && a[y] < a[y - 1] { // 3
-            a.swapAt(y - 1, y)
-            y -= 1
+    var sortedArray = array			 // 1
```

Comment:
Can we keep the `// 1`, `// 2`, etc vertical with each other?

Reply:
could you clarify what you mean by making them vertical